source/_patterns/00-atoms/02-typography/09-anker.mustache

```diff
@@ -0,0 +1,8 @@
+<a href="">
+  {{#ankerContent}}
+    {{ankerContent}}
+  {{/ankerContent}}
+  {{^ankerContent}}
+    A simple anker tag. It's clickable
+  {{/ankerContent}}
+</a>
```

Comment:
And the _ankers_ strike again! Please fix this.

Reply:
Same in https://github.com/tourstream/fti-portal-styleguide/pull/127#pullrequestreview-258282593

Reply:
You mean wording? If so, then done!